gadget/update.go

```diff
@@ -61,11 +61,16 @@ type ContentChange struct {
 }
 
 type ContentOperation int
+type ContentChangeDisposition int
 
 const (
 	ContentWrite ContentOperation = iota
 	ContentUpdate
 	ContentRollback
+
+	ChangeExecute ContentChangeDisposition = iota
+	ChangeAbort
```

Comment:
should this be the first value of the enum given that is used with errors to follow go patterns?